mapviz/launch/mapviz.launch.py

```diff
@@ -0,0 +1,40 @@
+import launch
+import launch.actions
+import launch.substitutions
+import launch_ros.actions
+
+def generate_launch_description():
+    return launch.LaunchDescription([
+        launch_ros.actions.Node(
+            package="mapviz",
+            node_executable="mapviz",
+            node_name="mapviz",
+        ),
+        launch_ros.actions.Node(
+            package="swri_transform_util",
+            node_executable="initialize_origin.py",
+            node_name="initialize_origin",
+            parameters=[
+                {"name": "local_xy_frame", "value": "map"},
+                {"name": "local_xy_origin", "value": "swri"},
+                {"name": "local_xy_origins", "value": [
+                    {"name": "swri",
```

Comment:
Do we want to have local lat/longs pushed into the public repo?

Reply:
That's a good question, although we've had that in there since the earliest versions of mapviz, and it's public information, so... :man_shrugging: